addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -497,17 +497,22 @@ class StudyService extends Service {
   }
 
   async list(requestContext, category, fields = []) {
```

Comment:
what is the behavior when during workspace provisioning an open data study is attempted to be attached that we have in DynamoDB but not included in the filters? That situation won't arise from the UI but someone can arbitrarily try to include any study from provision API directly.

Reply:
Updated the create workspace API to not allow users to create workspace with an Open Study that is not in the list of valid Open Studies.